README.md

```diff
@@ -26,6 +26,14 @@ Just execute:
 
     $ ./build.js
 
+In the occassion you are running a windows platform, we will have to make a few fixes for your machine to be able to build this app:
```

Comment:
typo: Windows. Also maybe "*you* will have", not "we will have"

Possibly add a section for this, too...